debian/jellyfin.init

```diff
@@ -0,0 +1,44 @@
+### BEGIN INIT INFO
+# Provides:          Jellyfin Media Server
+# Required-Start:    $local_fs $network
+# Required-Stop:     $local_fs
+# Default-Start:     2 3 4 5
+# Default-Stop:      0 1 6
+# Short-Description: Jellyfin Media Server
+# Description:       Runs Jellyfin Server
+### END INIT INFO
+
+# Carry out specific functions when asked to by the system
+pid=`ps -fA|grep dotnet|grep JellyfinServer|awk '{print $2}'| tr -d '\n'`
+
+case "$1" in
+  start)
+        if [ "$pid" == "" ]; then
+                echo "Starting Jellyfin..."
+                nohup dotnet /usr/lib/jellyfin/bin/EmbyServer.dll >/dev/null 2>&1 &
```

Comment:
Note that my script, as you seem to base on that, does not change the user, so it will run as root. This needs to be fixed sometime in the future I think.

Reply:
Yup will have to fix that as well.